<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-K28EKLKS" ignored="3" name="Test(failed)(failed)" tests="4" failures="1" timestamp="10 Jan 2020 07:04:56 GMT" time="0.096" errors="0">
  <testcase name="tCase2" time="0.027" classname="reRunner.RerunnerDemo">
    <skipped/>
  </testcase> <!-- tCase2 -->
  <testcase name="tCase2" time="0.0" classname="reRunner.RerunnerDemo">
    <skipped/>
  </testcase> <!-- tCase2 -->
  <testcase name="tCase2" time="0.003" classname="reRunner.RerunnerDemo">
    <skipped/>
  </testcase> <!-- tCase2 -->
  <testcase name="tCase2" time="0.002" classname="reRunner.RerunnerDemo">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at reRunner.RerunnerDemo.tCase2(RerunnerDemo.java:18)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tCase2 -->
  <testcase name="tCase1" time="0.0" classname="reRunner.RerunnerDemo">
    <ignored/>
  </testcase> <!-- tCase1 -->
  <testcase name="tCase3" time="0.0" classname="reRunner.RerunnerDemo">
    <ignored/>
  </testcase> <!-- tCase3 -->
  <testcase name="tCase4" time="0.0" classname="reRunner.RerunnerDemo">
    <ignored/>
  </testcase> <!-- tCase4 -->
</testsuite> <!-- Test(failed)(failed) -->
